controllers/storagecluster/storagecluster_controller.go

```diff
@@ -117,6 +119,31 @@ func (r *StorageClusterReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		},
 	}
 
+	enqueueStorageClusterRequest := handler.EnqueueRequestsFromMapFunc(
```

Comment:
To limit the number of re-queues (if/when we support multiple clusters), let's only list in the namespace where this ocsinit resource is present and only requeue for ocsinit that are valid, not the ones that are ignored or in error. i.e. `sc.namespace=ocsinit.namespace && ocsinit.status.phase==ready`

Reply:
done